### DISCUSSION TITLE
Can Someone Add a Icon For PBS Retro (1080p) [Geo-Blocked]

### DISCUSSION THREAD

**Discussion by Westh977** [iptv-org/community#1160] [Q&A]
https://static.wikia.nocookie.net/pbskids/images/d/de/PBS-Retro.webp/revision/latest?cb=20240425233022

**Comment by freearhey** [comment]
https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-edit-channel-description
